src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedFrameworkReference.xaml

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Rule Name="ResolvedFrameworkReference"
-      Description="Framework Reference Properties"
-      DisplayName="Framework Reference"
+      Description="Framework Properties"
```

Comment:
We call this "Reference Properties" everywhere else, why is Framework different?

Reply:
Not everywhere. This change was intended to match `ResolvedPackageReference`:

https://github.com/dotnet/project-system/blob/a1f2e6e0465fa481c9014c335b839ac59d2c357e/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedPackageReference.xaml#L3-L5

There is more we can do here. I filed #5203 to follow up.